### ISSUE TITLE
Multiple "invisible" tab key presses required to access "Add Split" button in "Split Translation" since v1.7.0

### ISSUE THREAD

**Issue by tomasz1986** [moneymanagerex/moneymanagerex#6405]
MMEX version:
 - [ ] 1.6.4
 - [x] 1.7.0
 - [ ] 1.7.1
 - [ ] 1.7.2
 - [ ] Other (please specify)

Operating System:
 - [x] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug

Before v1.7.0, it was possible to access the "Add Split" button using the `Tab` key on the keyboard directly after being focused on "Notes". However, since v1.7.0, it is required to press the `Tab` key **10 times** after being focused on Notes in order to move to the "Add Split" button. This doesn't seem normal, because nothing is visibly focused on the screen when pressing the button, yet you still need to press it 10 times until the "Add Split" button becomes focused.

![image](https://github.com/moneymanagerex/moneymanagerex/assets/5626656/80220fb2-dd70-4d37-b9a5-c2b01917c1e2)

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

1. Adding a new transaction, open the "Split Transaction" window.
2. Click on the first input field below "Category".
3. Keep pressing the `Tab` key in order to switch to next input fields.
4. Four `Tab`s should move the focus to "Notes".
5. From this moment, keep pressing the `Tab` key until you finally move the focus to "Add Split". As of v1.7.0, it takes exactly 10 `Tab`s to do it, while previously it required only a single `Tab`.

Expected result:

The keyboard focus should move immediately to the "Add Split" button after being focused on "Notes".

Actual result:

The user is forced to press the `Tab` key 10 times instead of 1 time in order to move to the "Add Split" button.

# Additional information

The issue began with v1.7.0 and I believe is likely related to the newly added "Tags" functionality. On a side note, not sure if intended, but in contrary to other fields, `Tags` doesn't use a bold font, and also the cursor indicator in the input field is narrower and uses different spacing than the cursor in the "Category" and "Amount" input fields.

**Comment by rmelillo76** [comment]
The label is deliberately not bold because Tags are not a required field (Bold labels are used as a visual clue to indicate the field is mandatory). We had some discussion on that during Tags development.

**Comment by whalley** [comment]
> However, since v1.7.0, it is required to press the Tab key 10 times after being focused on Notes in order to move to the "Add Split" button.

Yes, seems the 'Tags' field on the unavailable splits is being navigated to on each 'Tab'.

**Comment by tomasz1986** [comment]
The exact same bug is back in version v1.9.2. How has this happened? 😓

Is there a chance for a quick fix? This is an accessibility issue, making keyboard navigation very cumbersome.

**Comment by whalley** [comment]
> The exact same bug is back in version v1.9.2. How has this happened? 😓
> 
> Is there a chance for a quick fix? This is an accessibility issue, making keyboard navigation very cumbersome.

Can you please raise a new issue for this.

**Comment by tomasz1986** [comment]
Reported again in https://github.com/moneymanagerex/moneymanagerex/issues/8100.
